src/sdk/model/o365/o365-restore-session.ts

```diff
@@ -540,14 +556,19 @@ export class O365RestoreSession extends Entity {
   }
 
   /**
-   * Restore O365 SharePoint Site
-   * @param siteId
-   * @param request
+   * Restore O365 SharePoint Site given the restore session UUID
+   * its Veeam native id. This method will return TaskResponse object immediately
+   * and submits the restoration task in the background.
+   * @param {string} siteId
+   * @param {O365SharePointRestoreOptionsRequest} request
    */
```

Comment:
Add return type to doc. Only thing I saw.

Reply:
@SuperConfuserUser added! Thanks for the catch